packages/math/src/pool/concentrated/quotes.ts

```diff
@@ -0,0 +1,112 @@
+import { Coin, Dec, Int } from "@keplr-wallet/unit";
+
+import { maxSpotPrice, minSpotPrice, smallestDec } from "./const";
+import { TickOverflowError } from "./errors";
+import { makeSwapStrategy } from "./swap-strategy";
+import { sqrtPriceToTick, tickToSqrtPrice } from "./tick";
+import { addLiquidity, approxSqrt } from "./utils";
+
+export const ConcentratedLiquidityMath = {
+  calcOutGivenIn,
+  // calcInGivenOut,
```

Comment:
Please don't start on this one right now. I think it will be much easier to port once chain-side refactor is complete